### ISSUE TITLE
test 205: fails to create an ALT text

### ISSUE THREAD

**Issue by lvnilesh** [snarfed/wordpress-micropub#65]
https://micropub.rocks/server-tests/205?endpoint=107

205: Create an h-entry post with a photo with alt text (JSON)

I see the photo of a sunset but the associated alt text is `""`

```
<div id='gallery-2' class='gallery galleryid-9230 gallery-columns-1 gallery-size-full'><dl class='gallery-item'>
			<dt class='gallery-icon landscape'>
				<a href='https://be.a.cloudgeni.us/9230-2/photo-of-a-sunset-2/'><img width="1600" height="1200" src="https://be.a.cloudgeni.us/wp-content/uploads/sunset-9.jpg" class="attachment-full size-full" alt="" /></a>
			</dt></dl>
</div>
```

What am I missing?

**Comment by snarfed** [comment]
huh. i'm not sure what you're missing, but that test definitely passed on micropub.rocks last time we tried it with this plugin, and i doubt it's broken since then. https://micropub.rocks/implementation-reports/servers/27/Ouvg5VCJrclDWHUtQHXA (test # 13)

**Comment by lvnilesh** [comment]
Thanks. I will investigate. In the meanwhile, please see this https://micropub.rocks/implementation-reports/servers/107/QehJJW6izH3dmNKftbpK

**Comment by snarfed** [comment]
thanks! that report actually shows that the alt text test (# 13) is blank, which means it wasn't run. if it failed, it would show a red X, like test # 6 here: https://micropub.rocks/implementation-reports/servers/70/hQ8v0XfiLAjYgEZ6qvon

**Comment by lvnilesh** [comment]
I ran it. the last (verification step) is manual. I didn't click because I found alt=""

**Comment by dougbeal** [comment]
I ran into the same problem, the image ends up with a blank alt-tag

photo:

&gt; [](https://dougbeal.com/2018/07/17/1920/)

meta-data
`
KeyValue
micropub_auth_response'a:7:{s:10:"token_type";s:6:"Bearer";s:5:"scope";s:29:"create update delete undelete";s:2:"me";s:20:"https://dougbeal.com";s:9:"issued_by";s:48:"https://dougbeal.com/wp-json/indieauth/1.0/token";s:9:"client_id";s:23:"https://micropub.rocks/";s:9:"issued_at";i:1530145256;s:4:"user";i:2;}'
mf2_type'a:1:{i:0;s:7:"h-entry";}'
mf2_content'a:1:{i:0;s:112:"Micropub test of creating a photo referenced by URL with alt text. This post should include a photo of a sunset.";}'
mf2_photo'a:1:{i:0;a:2:{s:5:"value";s:39:"https://micropub.rocks/media/sunset.jpg";s:3:"alt";s:17:"Photo of a sunset";}}'
_mentionme'1'
_edit_lock'1531873209:2'
`
`
webserver-wordpress_1 | 2018/07/18 00:10:30 [notice] 5#5: *13454 "^/.well-known/(host-meta|webfinger).*" does not match "/", client: 173.230.155.197, server: dougbeal.com, request: "POST /?micropub=endpoint HTTP/1.1", host: "dougbeal.com"
[18-Jul-2018 00:10:30 UTC] Micropub Data: {"micropub":"endpoint"} {"type":["h-entry"],"properties":{"content":["Micropub test of creating a photo referenced by URL with alt text. This post should include a photo of a sunset."],"photo":[{"value":"https:\/\/micropub.rocks\/media\/sunset.jpg","alt":"Photo of a sunset"}]}}
[18-Jul-2018 00:10:30 UTC] wp_insert_post with args: {"post_content":"Micropub test of creating a photo referenced by URL with alt text. This post should include a photo of a sunset.","meta_input":{"micropub_auth_response":{"token_type":"Bearer","scope":"create update delete undelete","me":"https:\/\/dougbeal.com","issued_by":"https:\/\/dougbeal.com\/wp-json\/indieauth\/1.0\/token","client_id":"https:\/\/micropub.rocks\/","issued_at":1530145256,"user":2},"mf2_type":["h-entry"],"mf2_content":["Micropub test of creating a photo referenced by URL with alt text. This post should include a photo of a sunset."],"mf2_photo":[{"value":"https:\/\/micropub.rocks\/media\/sunset.jpg","alt":"Photo of a sunset"}]},"post_author":2,"post_status":"publish"}
webserver-wordpress_1 | 2018/07/18 00:10:30 [notice] 5#5: *13454 "^/.well-known/(host-meta|webfinger).*" does not match "/index.php", client: 173.230.155.197, server: dougbeal.com, request: "POST /?micropub=endpoint HTTP/1.1", host: "dougbeal.com"
webserver-wordpress_1 | 2018/07/18 00:10:30 [notice] 5#5: *13371 "^/.well-known/(host-meta|webfinger).*" does not match "/feed/", client: 66.249.90.227, server: dougbeal.com, request: "GET /feed/ HTTP/1.1", host: "dougbeal.com"
webserver-wordpress_1 | 2018/07/18 00:10:30 [notice] 5#5: *13371 "^/.well-known/(host-meta|webfinger).*" does not match "/index.php", client: 66.249.90.227, server: dougbeal.com, request: "GET /feed/ HTTP/1.1", host: "dougbeal.com"
webserver-wordpress_1 | 2018/07/18 00:10:30 [notice] 5#5: *13457 "^/.well-known/(host-meta|webfinger).*" does not match "/wp-cron.php", client: 172.18.0.1, server: dougbeal.com, request: "POST /wp-cron.php?doing_wp_cron=1531872630.9701049327850341796875 HTTP/1.1", host: "dougbeal.com", referrer: "https://dougbeal.com/wp-cron.php?doing_wp_cron=1531872630.9701049327850341796875"
wordpress_1 | 172.18.0.4 - 18/Jul/2018:00:10:30 +0000 "POST /wp-cron.php" 200
webserver-wordpress_1 | 172.18.0.1 - - [18/Jul/2018:00:10:31 +0000] "POST /wp-cron.php?doing_wp_cron=1531872630.9701049327850341796875 HTTP/1.1" 200 5 "https://dougbeal.com/wp-cron.php?doing_wp_cron=1531872630.9701049327850341796875" "WordPress/4.9.7; https://dougbeal.com" "-"
wordpress_1 | 172.18.0.4 - 18/Jul/2018:00:10:30 +0000 "GET /index.php" 200
`

(Originally published at: https://crw.moe/b/Y2)
